images/proftpd/entrypoint.sh

```diff
@@ -18,12 +18,35 @@ if [ -e /run/secrets/$FTPUSER_PASSWORD_SECRET ] && ! id -u "$FTPUSER_NAME"; then
     | chpasswd -e
 fi
 
+if [ "$SFTP_ENABLE" = "on" ]; then
+  mkdir /etc/ssh
+  ssh-keygen -m PEM -f /etc/ssh/ssh_host_rsa_key -N '' -t rsa -b 2048
+  ssh-keygen -m PEM -f /etc/ssh/ssh_host_dsa_key -N '' -t dsa -b 1024
+  ssh-keygen -m PEM -f /etc/ssh/ssh_host_ecdsa_key -N '' -t ecdsa -b 521
```

Comment:
Please add a conditional inside this section that checks for existence of the rsa key file, so the host keys don't change every time the container starts. Note also that the mkdir needs to have the "-p" option so it won't error out on restart.

Mention the need for a persistent volume mount at /etc/ssh in the README.md.